lua/mason-registry/init.lua

```diff
@@ -193,4 +193,23 @@ function M.refresh(cb)
     end
 end
 
+---table<string, string[]>
+local aliases = {}
+
+---Register aliases for the specified packages
+---@param new_aliases table<string, string[]>
+function M.register_aliases(new_aliases)
```

Comment:
Let's rename these functions to narrow it down to packages specifically (future proofing)

```suggestion
function M.register_package_aliases(new_aliases)
```

Reply:
done! I also renamed `get_aliases` to `get_package_aliases` to match